stackpulse/manifest.json

```
@@ -0,0 +1,38 @@
+{
+  "display_name": "StackPulse",
+  "maintainer": "support@stackpulse.io",
+  "manifest_version": "1.0.0",
+  "name": "stackpulse",
+  "metric_prefix": "stackpulse.",
```

```suggestion
  "metric_prefix": "",
```
No need to se it since there are no metrics on this integration